tests/acceptance/fixtures.py

```diff
@@ -579,23 +579,30 @@ def test_foo():
 
 
 @pytest.fixture(autouse=True)
-def only_with_distro_feature(request, bitbake_variables):
-    """Fixture that enables use of `only_with_distro_feature(feature1, feature2)` mark.
+def only_with_mender_feature(request, bitbake_variables):
+    """Fixture that enables use of `only_with_mender_feature(feature1, feature2)` mark.
```

Comment:
Reminder (because I see you just force-pushed)

We agreed on the call to leave `only_with_distro_feature` fixture as-is, to be used by old branches of meta-mender and mender-convert, and just create a new one `only_with_mender_feature` for branches of meta-mender that actually have changed this (master, dunfell)

Reply:
Disregard, I completely miss read your code